### DISCUSSION TITLE
zksynk testnet deploy remix

### DISCUSSION THREAD

**Discussion by patricenewbie** [Cyfrin/foundry-full-course-cu#2672] [Q&A]
can anyone can help me? 
i try to deploy my contract from remix to the network zksynk sepolia testnet and i have this issue:
Gas estimation failed
Gas estimation errored with the following message (see below). The transaction execution will likely fail. Do you want to force sending?
Returned error: Failed to serialize transaction: toAddressIsNull
[object Object]

I have absolutely no idea how to solve this problem

here my contract:
pragma solidity ^0.8.18;

contract HelloWorld {
    string public greet = "Hello, zkSync!";
}
in metamask i have configured the following network for zksynk:

 URL da RPC : https://sepolia.era.zksync.dev

ID: 300

symbol: ETH

**Comment by EngrPips** [comment]
Hello @patricenewbie, `ZkSync` part of the course is optional, so you should proceed if working with `ZkSync` seems to be difficult.
